tests/rptest/tests/franz_go_verifiable_test.py

```diff
@@ -83,3 +86,123 @@ def test_with_all_type_of_loads(self):
                                         start_timeout=200,
                                         stop_timeout=200)
             time.sleep(30)
+
+
+class SiWithFranzGoTest(RedpandaTest):
+    MSG_SIZE = 120000
+
+    segment_size = 16 * 1048576
+    s3_host_name = "minio-s3"
+    s3_access_key = "panda-user"
+    s3_secret_key = "panda-secret"
+    s3_region = "panda-region"
+    s3_topic_name = "panda-topic"
+
+    topics = (TopicSpec(partition_count=1000, replication_factor=3), )
```

Comment:
is this test appropriate for local dev runs? the default allocation size in the segment appender may need to be updated so this test doesn't eat up 32 GB of disk on start up. But perhaps there are other parts of the test that are also assuming lots of resources.

either way is ok. we need big tests! but we might want to parameterize it so we can go smaller or even bigger.